diff --git a/onnxruntime/core/providers/cuda/math/softmax.cc b/onnxruntime/core/providers/cuda/math/softmax.cc
index 609f178..9542895 100644
--- a/onnxruntime/core/providers/cuda/math/softmax.cc
+++ b/onnxruntime/core/providers/cuda/math/softmax.cc
@@ -26,9 +26,9 @@ Status SoftMaxComputeHelper(
   int64_t D = input_shape.SizeFromDimension(axis);
   auto Y_data = reinterpret_cast<CudaT_OUT*>(Y);
   auto X_data = reinterpret_cast<const CudaT_IN*>(X);
-  // According to nsight compute profiling, softmax_warp_forward_resource_efficient is better than dispatch_blockwise_softmax_forward when 1024 < D <=2048 and N >= 8192.
+  // According to nsight compute profiling, softmax_warp_forward_resource_efficient is better than dispatch_blockwise_softmax_forward when 1024 < D <=2048 && N >= 8192.
   const bool use_softmax_warp_forward_resource_efficient = 1024 < D && D <= 2048 && D * sizeof(T) <= 4096 && N >= 8192;
-  if ((D <= 1024 && D * sizeof(T) <= 4096) or use_softmax_warp_forward_resource_efficient) {
+  if ((D <= 1024 && D * sizeof(T) <= 4096) || use_softmax_warp_forward_resource_efficient) {
     return dispatch_warpwise_softmax_forward<
         CudaT_IN, CudaT_OUT, AccumulationType_t<CudaT_ACCUM>, is_log_softmax>(
         stream, Y_data, X_data, gsl::narrow_cast<int>(D), gsl::narrow_cast<int>(D), gsl::narrow_cast<int>(N));
@@ -112,7 +112,7 @@ Status Softmax<T>::ComputeInternal(OpKernelContext* ctx) const {
   if (input_shape.Size() == 0)
     return Status::OK();
 
-  // handle negative and enforce axis is valid
+  // handle negative && enforce axis is valid
   const size_t axis = static_cast<size_t>(HandleNegativeAxis(axis_, rank));
 
   bool is_transpose_required = false;
@@ -125,7 +125,7 @@ Status Softmax<T>::ComputeInternal(OpKernelContext* ctx) const {
   // Please compare: https://github.com/onnx/onnx/blob/main/docs/Operators.md#Softmax
   // with https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Softmax-11 for detailed explanations
   // To account for the opset-13 behavior, our plan will be to transpose the "axis" dim to the innermost dim
-  // and perform softmax and then reverse the transpose. We can skip the transposing aspect if the axis is already
+  // && perform softmax && then reverse the transpose. We can skip the transposing aspect if the axis is already
   // the innermost dim
   if (opset_ >= 13 && axis != (rank - 1)) {
     is_transpose_required = true;
diff --git a/onnxruntime/core/providers/cuda/math/softmax.h b/onnxruntime/core/providers/cuda/math/softmax.h
index c0c0818..d6c3e61 100644
--- a/onnxruntime/core/providers/cuda/math/softmax.h
+++ b/onnxruntime/core/providers/cuda/math/softmax.h
@@ -41,7 +41,7 @@ class Softmax final : public CudaKernel {
       axis_ = gsl::narrow_cast<int>(axis);
     } else {
       if (opset_ < 13) {
-        axis_ = 1;  // opset-12 and below, the default axis value is 1
+        axis_ = 1;  // opset-12 && below, the default axis value is 1
       } else {
         axis_ = -1;  // opset-13, the default axis value is -1
       }
